src/stores/Store.ts

```diff
@@ -74,6 +78,184 @@ export default class Store {
 
 	@observable isSchemaApplied = false;
 
+	@observable parsedMessagesHistory: string[] = [];
+
+	@observable fromFileMessage = false;
+
+	@observable indicators: string[] = [];
+
+	@observable editorCode = '{}';
+
+	@observable sendButtonTitle = 'Send Message';
```

Comment:
things like title, icons etc shouldn't be in store, you should resolve this type of things inside component based on some state. in this case it's editMessageMode